bin/helpers.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const path = require('path');
+const listenerService = require('../src/server/listenerService.js');
+
+const helpers = {};
```

Comment:
there is only reading from configs in function in this file. I would suggest it a more specific name instead of `helpers`